e2e/atlas/events_test.go

```diff
@@ -64,6 +65,7 @@ func TestEvents(t *testing.T) {
 		cmd := exec.Command(cliPath,
 			atlasEntity,
 			eventsEntity,
+			"organization",
```

Comment:
you can add `organizationEntity` in https://github.com/mongodb/mongocli/blob/6ac1f831b2a7d4ae60f4a7d025e4ab3c8da50d79/e2e/atlas/helper_test.go#L46 as we have for the project and then use it in your test